### PR TITLE
Prepare for 0.15.1

### PR DESCRIPTION
**Pull request by meyskens**
Prepare for  0.15.1 release

### CODE REVIEW

File: bundle/cert-manager-operator/0.15.0-patch.1/cert-manager-operator.v0.15.0-patch.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    description: "x509 certificate management for Kubernetes"
+    createdAt: "2020-05-04T12:00:00Z"
+    support: "cert-manager team"
+    alm-examples: ' [{"apiVersion":"operator.cert-manager.io/v1alpha1","kind":"CertManager","metadata":{"name":"cert-manager","namespace":"cert-manager"},"spec":{}}]'
+    capabilities: Full Lifecycle
+    categories: Security
+    certified: 'true'
+    containerImage: registry.connect.redhat.com/jetstack/cert-manager-operator:v0.15.0-ubi.1
+    repository: https://github.com/jetstack/cert-manager
+  name: cert-manager-operator.v0.15.0-patch.1
```
Comment (munnerz) [inline bundle/cert-manager-operator/0.15.0-patch.1/cert-manager-operator.v0.15.0-patch.1.clusterserviceversion.yaml:14]: I don't think this is correct? 🤔

File: bundle/cert-manager-operator/0.15.0-patch.1/certmanagers.operator.cert-manager.io.crd.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```
Comment (munnerz) [inline bundle/cert-manager-operator/0.15.0-patch.1/certmanagers.operator.cert-manager.io.crd.yaml:1]: Don't think this file should be in this PR

File: bundle/cert-manager-operator/0.15.0-patch.1/cert-manager-operator.v0.15.0-patch.1.clusterserviceversion.yaml (hunk 1 above)
Comment (meyskens) [reply bundle/cert-manager-operator/0.15.0-patch.1/cert-manager-operator.v0.15.0-patch.1.clusterserviceversion.yaml:14]: this is the patch version which I never pushed to GitHub however it is pushed to RedHat connect so we should keep a record of it.

File: bundle/cert-manager-operator/0.15.0-patch.1/certmanagers.operator.cert-manager.io.crd.yaml (hunk 2 above)
Comment (wallrj) [reply bundle/cert-manager-operator/0.15.0-patch.1/certmanagers.operator.cert-manager.io.crd.yaml:1]: @meyskens Is this here for the same reason as the clusterserviceversion.yaml (above) ?

**Comment by meyskens** [comment]
Good catch @wallrj updated it

**Pull request merged**